CustomScript/shim.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# The shim scripts provide a single entry point for CSE and will invoke the customscript.py entry point using the
+# appropriate python interpreter version.
+# Arguments passed to the shim layer are redirected to the invoked script without any validation.
+
+COMMAND="./customscript.py"
+PYTHON=""
+ARG="$@"
+
+function find_python(){
```

Comment:
I just cleaned up the [shim ](https://github.com/Azure/azure-linux-extensions/pull/649/files#diff-8e0deab0c440b69b28228e76a0fdf02fR34). You should try the cleaner `find_python`.

Just FYI - It also supports commands directly now.
```
$ bash extension_shim.sh -c ""hello.py --install""
python hello.py --install
```

Reply:
Done! Thanks for the help with this script!